specification/storagepool/resource-manager/Microsoft.StoragePool/preview/2021-04-01-preview/examples/IscsiTargets_Get.json

```diff
@@ -26,7 +26,8 @@
           "luns": [
             {
               "managedDiskAzureResourceId": "/subscriptions/11111111-1111-1111-1111-111111111111/resourceGroups/myResourceGroup/providers/Microsoft.Compute/disks/vm-name_DataDisk_1",
-              "name": "lun0"
+              "name": "lun0",
+              "lun": 0
```

Comment:
Do you want to add this property to the `IscsiTargets_List.json` file too?

Reply:
Updated all example files.